pkg/component/extensions/worker/worker.go

```diff
@@ -225,6 +225,11 @@ func (w *worker) deploy(ctx context.Context, operation string) (extensionsv1alph
 			}
 		}
 
+		var oscHash *string
+		if oscConfig.Init.IncludeHashInWorker {
+			oscHash = &oscConfig.Init.GardenerNodeAgentSecretName
```

Comment:
I would have expected a short comment here stating why a secret name can be used as hash in this context.

Reply:
The variable is now called `nodeAgentSecretName`.